avro-builder/src/main/java/com/linkedin/avroutil1/builder/operations/codegen/own/AvroUtilCodeGenOp.java

```diff
@@ -52,29 +51,10 @@ public void run(OperationContext opContext) throws Exception {
       return;
     }
     LOGGER.info("found " + avscFiles.size() + " avsc schema files");
+    generator.setInputs(config.getInputRoots());
 
```

Comment:
i dislike the idea of bumdling dircovery of schema files, parsing of schema files and (in the future) classpath lookup of referenced schemas into generator. i'd rather keep generator to be "just" the code-gen, and be able to handle these separately

Reply:
also in the future we'd be running validations on these schemas before generating them.

Reply:
Agreed. I can unbundle discovery, parsing, and code-gen in another PR. cc @li-ukumar in case you have thoughts here